src/libraries/Microsoft.Extensions.Diagnostics/src/Metrics/DebugConsoleMetricListener.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.Extensions.Diagnostics.Metrics
+{
+    internal sealed class DebugConsoleMetricListener : IMetricsListener, IDisposable
+    {
+        private readonly Timer _timer;
+        internal TextWriter _textWriter = Console.Out;
```

Comment:
>internal TextWriter _textWriter = Console.Out; [](http://example.com/codeflow?start=8&length=46)

what happen if the console output get redirected after creating DebugConsoleMetricListener object?